code/handlers/trackservers.q

```diff
@@ -335,4 +335,19 @@ pc:{[result;W] update w:0Ni,endp:.proc.cp[] from`.servers.SERVERS where w=W;clea
 if[enabled;
 	.z.pc:{.servers.pc[x y;y]}.z.pc;
 	if[DISCOVERYRETRY > 0; .timer.repeat[.proc.cp[];0Wp;DISCOVERYRETRY;(`.servers.retrydiscovery;`);"Attempt reconnections to the discovery service"]];
-	if[RETRY > 0; .timer.repeat[.proc.cp[];0Wp;RETRY;(`.servers.retry;`);"Attempt reconnections to closed server handles"]]];
+    if[RETRY > 0; .timer.repeat[.proc.cp[];0Wp;RETRY;(`.servers.retry;`);"Attempt reconnections to closed server handles"]]];
```

Comment:
Keep indentation consistent with above, may need to use tabs or changes the others to spaces.